### PR TITLE
fixed doxygen in g2_getfld.c 

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #73
Part of #54

fixed doxygen in g2_getfld.c

### CODE REVIEW

File: src/g2_gribend.c [hunk 1]
```diff
@@ -1,51 +1,38 @@
...
+ * create a complete GRIB2 message. g2_create() must be called first to
+ * initialize a new GRIB2 message.
+ *
+ * @param cgrib Char array containing all the data sections added be
+ * previous calls to g2_create, g2_addlocal, g2_addgrid, and
+ * g2_addfield. After function is called, contains the finalized GRIB2
+ * message.
+ *
+ * @return
+ * - > 0 = Length of the final GRIB2 message in bytes.
```
Comment (Hang-Lei-NOAA) [inline src/g2_gribend.c:24]: Should this '>' be removed ?

Comment (edwardhartnett) [reply src/g2_gribend.c:24]: No I thnk its correct...

**Comment by Hang-Lei-NOAA** [comment]
Why some return add - ">" but others not? What is the difference?

**Review by Hang-Lei-NOAA — APPROVED** [review]

**Comment by edwardhartnett** [comment]
In this case it means greater than, so:

 - > 0 = Length of the final GRIB2 message in bytes.
 
Means it it returns a value > 0, it is the length of the message.

**Pull request merged**